LFS171x/docs/introduction-to-hyperledger.md

```diff
@@ -1 +1,386 @@
+	- [Interoperability between Hyperledger Frameworks](#interoperability-between-hyperledger-frameworks)
+- [Knowledge Check](#knowledge-check)
+	- [Knowledge Check 2.1](#knowledge-check-21)
+	- [Knowledge Check 2.2](#knowledge-check-22)
+	- [Knowledge Check 2.3](#knowledge-check-23)
+	- [Knowledge Check 2.4](#knowledge-check-24)
+- [Conclusions & Learning Objectives (Review)](#conclusions-learning-objectives-review)
+	- [Learning Objectives (Review)](#learning-objectives-review)
+	- [Blockchain Security at Hyperledger (David Huseby)](#blockchain-security-at-hyperledger-david-huseby)
+	- [How Will Hyperledger Change the Blockchain Ecosystem? (Brian Behlendorf)](#how-will-hyperledger-change-the-blockchain-ecosystem-brian-behlendorf)
+
+<!-- /TOC -->
+
+
 # Introduction and Learning Objectives
+
+## Introduction
+This chapter provides an overview of Hyperledger, a collaborative project hosted by The Linux Foundation that is focused on business blockchain technologies. It also introduces the current Hyperledger frameworks and modules, as of October 2017.
+
+## Learning Objectives
+By the end of this chapter you should be able to:
+
+* Explain the differences between Hyperledger and permisionless blockchain technologies.
+* Discuss how Hyperledger leverages open standards and open governance to support business solutions.
+* Discuss Hyperledger frameworks (Iroha, Sawtooth, Fabric, Indy, and Burrow) and modules (Cello, Explorer, and Composer).
+
+# Hyperledger
+
+## Hyperledger (Navroop Sahdev)
+[![Hyperledger (Navroop Sahdev)](../images/video-image.png)](https://youtu.be/sE5eMx7TTWM)
+
+## Hyperledger
+Hyperledger is a group of open source projects focused around cross-industry distributed ledger technologies. Hosted by The Linux Foundation, collaborators include industry leaders in technology, finance, banking, supply chain management, manufacturing, and IoT.
+
+As of October 2017, Hyperledger consists of eight projects, five of which are distributed ledger frameworks. The other three projects are modules that support these frameworks.
+
+![Hyperledger Modular Umbrella Approach](../images/introduction-to-hyperledger/modular_umbrella.jpg)
+
+As [Arnaud Le Hors](https://www.hyperledger.org/blog/2017/09/12/3431), member of the Hyperledger Technical Steering Committee, emphasized,
+
+>"these projects show how broadly applicable blockchain technology really is. This goes way beyond cryptocurrencies".
+
+[Hyperledger](https://www.hyperledger.org/about) provides an alternative to the cryptocurrency-based blockchain model, and focuses on developing blockchain frameworks and modules to support global enterprise solutions. The focus of Hyperledger is to provide a transparent and collaborative approach to blockchain development.
+
+## The Birth of Hyperledger (Brian Behlendorf)
+[![The Birth of Hyperledger (Brian Behlendorf)](../images/video-image.png)](https://youtu.be/iNVT2Pdy--k)
+[Some rights reserved](https://creativecommons.org/licenses/by-nc-nd/4.0/)
```

Comment:
Remove - the course is actually licensed under https://creativecommons.org/licenses/by/4.0/. Unfortunately, EdX doesn't let you specify which Creative Common license for videos. Therefore, we will let the overall license cover these.